scraper/.gitignore

```diff
@@ -0,0 +1 @@
+config.ini
```

Comment:
nit: no newline at the end of the file.

Reply:
fixed in https://github.com/230443/Sprinters/pull/15/commits/832085a3c3d729274b6b57a6662e134c1027af6a